### ISSUE TITLE
Overhaul documentation

### ISSUE THREAD

**Issue by Niek** [Xetera/ghost-cursor#13]
The README needs some updating to reflect the changes in the code recently:
* Example of `installMouseHelper`
* New function signature for `createCursor`
* New function signature for `move` and `click`
* Document `moveTo` and `getRandomPagePoint`

Anybody who's willing to check this?

**Comment by Niek** [comment]
Sample TS code:
```typescript
import * as puppeteer from 'puppeteer'
import { createCursor, getRandomPagePoint, installMouseHelper } from 'ghost-cursor'

(async () => {
  const browser = await puppeteer.launch()
  const page = await browser.newPage()
  const cursor = createCursor(page, await getRandomPagePoint(page), true)
  await installMouseHelper(page) // this shows the trace
  page.goto('https://www.example.com/')
  await cursor.click('element', { waitForClick: 10000, waitForSelector: 1000, paddingPercentage: 20 })
})()

```

**Comment by Somberor** [comment]
@Niek how can the speed of the mouse movement be set?
Sometimes it goes too fast and I would like it to have some minimum
Thank you for the great implementation!

**Comment by Niek** [comment]
> @Niek how can the speed of the mouse movement be set?
> Sometimes it goes too fast and I would like it to have some minimum
> Thank you for the great implementation!

This can be changed in the `path()` function - if you want to add speed customization, feel free to open a PR.

**Comment by clouedoc** [comment]
I'd like to add an item to the list:

- [ ] Document the fact that the cursor supports XPath

**Comment by TheGP** [comment]
I generated some docs with ChatGPT, wanna see what you'll think :)

**Comment by Niek** [comment]
Fixed in #116

**Comment by rublev** [comment]
> Sample TS code:
> 
> ```ts
> import * as puppeteer from 'puppeteer'
> import { createCursor, getRandomPagePoint, installMouseHelper } from 'ghost-cursor'
> 
> (async () => {
>   const browser = await puppeteer.launch()
>   const page = await browser.newPage()
>   const cursor = createCursor(page, await getRandomPagePoint(page), true)
>   await installMouseHelper(page) // this shows the trace
>   page.goto('https://www.example.com/')
>   await cursor.click('element', { waitForClick: 10000, waitForSelector: 1000, paddingPercentage: 20 })
> })()
> ```

file.js:4
  installMouseHelper,
  ^^^^^^^^^^^^^^^^^^
SyntaxError: Named export 'installMouseHelper' not found.
